tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt

```diff
@@ -20,7 +20,11 @@
 %L = (long|short|narrow)
 %M = (Alaska_Hawaii|Bering|Dominican|Goose_Bay|Greenland_Central|Dutch_Guiana|Africa_FarWestern|Liberia|British|Irish|Kuybyshev|Sverdlovsk|Baku|Tbilisi|Turkey|Yerevan|Aktyubinsk|Ashkhabad|Dushanbe|Frunze|Kizilorda|Oral|Samarkand|Shevchenko|Tashkent|Uralsk|Urumqi|Dacca|Karachi|Borneo|Malaya|Kwajalein)
 %N = (gregorian|generic|buddhist|chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|japanese|persian|roc)
-%O = (gregorian|chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|persian)
+# calendar systems that should use Gregorian months (just Gregorian)
+# We don't list roc, etc here because their months are hidden.
+%G = (gregorian)
```

Comment:
This should support not only gregorian, but also gregorian-based calendars. (Those that use the same 12 month solar calendar: Thai Buddhist, Julian, etc.

Reply:
Here are the calendars. Unfortunately, we don't have data here as to which is gregorian-based.
```
            <type name="buddhist" description="Thai Buddhist calendar"/>
            <type name="chinese" description="Traditional Chinese calendar"/>
            <type name="coptic" description="Coptic calendar"/>
            <type name="dangi" description="Traditional Korean calendar" since="22.1"/>
            <type name="ethioaa" description="Ethiopic calendar, Amete Alem (epoch approx. 5493 B.C.E)" alias="ethiopic-amete-alem"/>
            <type name="ethiopic" description="Ethiopic calendar, Amete Mihret (epoch approx, 8 C.E.)"/>
            <type name="gregory" description="Gregorian calendar" alias="gregorian"/>
            <type name="hebrew" description="Traditional Hebrew calendar"/>
            <type name="indian" description="Indian calendar"/>
            <type name="islamic" description="Hijri calendar"/>
            <type name="islamic-umalqura" description="Hijri calendar, Umm al-Qura" since="24"/>
            <type name="islamic-tbla" description="Hijri calendar, tabular (intercalary years [2,5,7,10,13,16,18,21,24,26,29] - astronomical epoch)" since="24"/>
            <type name="islamic-civil" description="Hijri calendar, tabular (intercalary years [2,5,7,10,13,16,18,21,24,26,29] - civil epoch)" since="24"/>
            <type name="islamic-rgsa" description="Hijri calendar, Saudi Arabia sighting" since="24"/>
            <type name="iso8601" description="ISO calendar (Gregorian calendar using the ISO 8601 calendar week rules)" since="2.0"/>
            <type name="japanese" description="Japanese Imperial calendar"/>
            <type name="persian" description="Persian calendar"/>
            <type name="roc" description="Republic of China calendar"/>
            <type name="islamicc" description="Civil (algorithmic) Arabic calendar" deprecated="true" preferred="islamic-civil"  alias="islamic-civil"/>
```

Reply:
I ran a quick check, and it appears that the following are gregorian-based: 
[gregoriy, iso8601, buddhist, japanese, roc]

Also filed a ticket to add metadata to cldr, so that this can be unhardcoded

Reply:
So actually it seems the "intent" was to hide those so that you don't have a Japanese month January with a different translation than Gregorian January. That mystified me too but I think that's why %G is just one and not the others

Reply:
@pedberg-icu should also weigh in on this.

- It clearly should not be limited to exactly 'gregory', since iso8601 uses the same months, etc.
- In the case of thai buddhist, japanese, roc, M2, M2 etc seem ok.

Reply:
If it's here, then it's visible for editing.  Do we want users giving a different translation for January in 8601 than for gregorian?

Reply:
@macchiati Specifically, this PR does not make any changes to the visibility (HIDE) or textual content for these calendars: `iso8601, buddhist, japanese, roc`. 

All that it does is change the following calendars to no longer have month codes based on Gregorian:

`chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|persian`